packages/office-ui-fabric-react/src/components/Coachmark/Coachmark.types.ts

```diff
@@ -128,6 +128,7 @@ export interface ICoachmarkProps extends React.Props<Coachmark> {
 
   /**
    * Ref for TeachingBubble
+   * @deprecated Coachmark uses focusFirstChild utility instead to focus on TeachingBubbleContent
```

Comment:
Should there be a `_warnDeprecations` call added for this deprecation?

Reply:
Added (as well as the other deprecated props)